ci-operator/jobs/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master-presubmits.yaml

```diff
@@ -50,6 +50,81 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )build,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
would you like it to run for every PR in capi repo?

Reply:
I want a periodic job... but I don't mind starting with something, seeing it pass few times and later adding the periodic.
I'll update the presubmit when I add the periodic